### ISSUE TITLE
smplx shape dimension is 16 not 10

### ISSUE THREAD

**Issue by Gaozhongpai** [mks0601/NeuralAnnot_RELEASE#18]
Hi Gyeongsik,

I ran the `demo_smplx.py` and it failed because the shape size in _smplx_ model is 16 while the data from the json file is 10. I added the following code to solve the issue:

`shape = torch.cat([shape, torch.zeros((1, 6))], dim=-1)`

Is this inconsistency caused by a different version of _smplx_ repository?

**Comment by mks0601** [comment]
Hi, sorry for the late reply. Maybe you're right, or you can explicitly set num_betas=10 when initializing smplx layer.
